src/test/java/io/whitesource/cure/SecureDeserializationTest.java

```diff
@@ -0,0 +1,57 @@
+package io.whitesource.cure;
```

Comment:
Loved all the hard work you put into mimic and mocking those classes, but unfortunately it's bad practice
Please checkout @Mockito
Look at this for more information: https://www.baeldung.com/mockito-series